.gitignore

```diff
@@ -8,3 +8,12 @@ docs/build/
 .cache
 .DS_Store
 .vscode
+.idea/
+htmldocs/
+*.exe
+*.dll
+*.orig
+*.bak
+*.bck
+* - Copy.*
+* - Copy (?).*
```

Comment:
Is that valid syntax? Wouldn't it be something like `*\ -\ Copy.*`?

Reply:
I believe it is (and it does work on Windows). According to `gitignore`'s documentation:

> Trailing spaces are ignored unless they are quoted with backslash ("\").

As far as I can tell that is the only place where you need to add backslashes.